addons/medical_feedback/functions/fnc_effectBloodVolumeIcon.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles the blood volume icon
+ *
+ * Arguments:
+ * 0: Enable <BOOL>
+ * 1: Intensity 0...6 <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true, 4] call ace_medical_feedback_fnc_effectBloodVolumeIcon
+ *
+ * Public: No
+ */
+
+params ["_enable", "_intensity"];
+
+if (!_enable || !GVAR(showBloodVolumeIcon)) exitWith {
+    with uiNamespace do {
+        if (ctrlText BloodVolumeInfoControlGroup == "") exitWith {};
+        BloodVolumeInfoControlGroup ctrlSetText "";
+        BloodVolumeInfoControlGroup ctrlSetTextColor ICON_BLOODVOLUME_COLOR_NONE;
+        BloodVolumeInfoControlGroup ctrlCommit 0;
+    };
```

Comment:
Don't get why this uses `with`. Just use getVariable.

Reply:
Removed with uiNamespace usage.